README.md

```diff
@@ -210,18 +210,17 @@ assert(mock.toString() == expected);
 ### Custom argument comparator
 
 You can provide a function, which will be used to compare two objects of a
-specific type. Use `configure` instead of the `Mocker` to create a custom mocker
-instance. `configure` takes a tuple of `Comparator`s, so you can specify as many
+specific type. Use `Configure` instead of the `Mocker` to create a custom mocker
+instance. `Configure` takes a tuple of “Comparators”, so you can specify as many
```

Comment:
Typographic quotes should be rendered automatically: just use `"` in Markdown.

BTW: without word wrapping in Markdown, chances for a minimal diff are better.
Break the lines at latest after the period.